sites/partners/pages/users/index.tsx

```diff
@@ -71,12 +80,20 @@ const Users = () => {
   const [itemsPerPage, setItemsPerPage] = useState<number>(AG_PER_PAGE_OPTIONS[0])
   const [currentPage, setCurrentPage] = useState<number>(1)
 
+  /* Add user drawer */
+  const [isDrawerOpen, setDrawerOpen] = useState<boolean>(false)
+
   /* Fetch user list */
   const { data: userList } = useUserList({
     page: currentPage,
     limit: itemsPerPage,
   })
 
+  /* Fetch listings */
```

Comment:
@seanmalbert Might eventually be worth a new query that only retrieves name, jurisdiction, and ID